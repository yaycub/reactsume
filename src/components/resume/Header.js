import React from 'react';
import PropTypes from 'prop-types';

const Header = ({ name = 'Cool Person', title = 'Super' }) => {
  return (
    <header>
      <h1>{name}</h1>
      <h2>{title}</h2>
    </header>
  );
};

Header.propTypes = {
  name: PropTypes.string,
  title: PropTypes.string
};

export default Header;
