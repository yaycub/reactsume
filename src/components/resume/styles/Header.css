.Header{
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  margin: 0 auto;
  width: 55vw;
  height: 15vh;
  padding: 20px 20px 20px 0px;

  h1{
    font-size: 45px;
    font-weight: bold;
    margin-bottom: 20px;
    
  }

  h2{
    font-size: 20px;
    font-style: italic;
    line-height: 30px;
    border-bottom: 5px solid black;
  }
}