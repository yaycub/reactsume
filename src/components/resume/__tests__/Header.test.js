import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header Component', () =>{
  it('can render a header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
