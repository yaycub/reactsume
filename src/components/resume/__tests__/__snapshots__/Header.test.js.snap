// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component can render a header 1`] = `
<header>
  <h1>
    Cool Person
  </h1>
  <h2>
    Super
  </h2>
</header>
`;
